src/auth0/handlers/clients.js

```diff
@@ -36,18 +36,24 @@ export default class ClientHandler extends DefaultHandler {
     // Do nothing if not set
     if (!clients) return;
 
+    const excludedClients = (assets.exclude && assets.exclude.clients) || [];
+
     const {
       del, update, create, conflicts
     } = await this.calcChanges(assets);
 
     // Always filter out the client we are using to access Auth0 Management API
     // As it could cause problems if it gets deleted or updated etc
     const currentClient = this.config('AUTH0_CLIENT_ID');
+
+    const filterClients = list => list
+      .filter(item => item.client_id !== currentClient && !excludedClients.includes(item.name));
```

Comment:
this _might_ be more efficient if we check `excludedClients` first?
```
    const filterClients = list => excludedClients.length ? list
      .filter(item => item.client_id !== currentClient && !excludedClients.includes(item.name)) : list;
```

Reply:
We're filtering `currentClient` anyway. So it would be like:
```? list.filter(item => item.client_id !== currentClient && !excludedClients.includes(item.name))
 : list.filter(item => item.client_id !== currentClient);```